extensions/HuggingFace/typescript/hf.ts

```diff
@@ -294,7 +293,7 @@ function constructOutput(response: TextGenerationOutput): Output {
 
   const output = {
     output_type: "execute_result",
-    data: data,
+    data: data.generated_text,
```

Comment:
Is there anything else that's in `data` other than `generated_text`?

It may make sense to put the raw response in the metadata, perhaps as a `rawResponse` property

Reply:
There's nothing else